test/e2e/serial/tests/configuration.go

```diff
@@ -693,9 +697,215 @@ var _ = Describe("[serial][disruptive] numaresources configuration management",
 				return kcCmNamesCur
 			}).WithContext(ctx).WithTimeout(5 * time.Minute).WithPolling(10 * time.Second).Should(Equal(kcCmNamesPre))
 		})
+
+		It("[reboot_required][slow][unsched][schedrst] should be able to correctly identify topology manager policy without scheduler restarting", Label("reboot_required", "slow", "unsched", "schedrst"), func(ctx context.Context) {
```

Comment:
for reboot tests we need to add an additional flag at the beginning of the test to avoid falling on NRT mismatch after reboot. this is a limitation in which kernel might distribute the resources (mainly memory) differently after node reboot. see here for example: https://github.com/openshift-kni/numaresources-operator/blob/main/test/e2e/serial/tests/configuration.go#L109

Reply:
got it ! added it